CONTRIBUTING.adoc

```diff
@@ -93,7 +93,14 @@ NOTE: Replace "linux/amd64" with the appropriate value for your platform/archite
 
         $ sudo chmod +r openshift.local.config/master/openshift-registry.kubeconfig
         $ sudo chmod +r openshift.local.config/master/admin.kubeconfig
-        $ oadm registry --create --credentials=openshift.local.config/master/openshift-registry.kubeconfig --config=openshift.local.config/master/admin.kubeconfig
+        $ sudo ./oadm registry --create --credentials=openshift.local.config/master/openshift-registry.kubeconfig --config=openshift.local.config/master/admin.kubeconfig
```

Comment:
Yup , those lines generally should do this.  But oddly I've seen the sudo incantation used in many other setups for various other reasons..... I thought it might be idiomatic to run oadm as root. 

I'm happy to close this for now